src/webapp/cilogon_ldap.py

```diff
@@ -66,10 +66,65 @@ def cilogon_id_map_to_yaml_data(m):
     return data
 
 
+def get_osgid_lookup(yaml_data):
+    osgid_lookup = {}
+    for id_, contact in yaml_data.items():
+        if 'CILogonID' in contact:
+            osgid_lookup[contact['CILogonID']] = contact
```

Comment:
```suggestion
    for contact in yaml_data.values():
        if 'CILogonID' in contact:
            osgid_lookup[contact['CILogonID']] = contact
```

Reply:
Yeah... The first time through i wasn't sure if i'd use the id - after refactoring to its current form clearly it is unused.  I noticed this after making the PR, and thought of making the same change, but was too lazy to change it yesterday :)